src/drive/locales/en.json

```diff
@@ -31,6 +31,9 @@
     "empty_trash": "Empty trash",
     "share": "Share"
   },
+  "intent": {
```

Comment:
I am not sure that "intent" should be in the locale key, it's providing an technical information about how we implement the feature. IMHO locale keys should only describe functionnal informations.

Reply:
I would suggest `service` here